### ISSUE TITLE
feat: Add contributor form to website

### ISSUE THREAD

**Issue by madiedgar** [legesher/legesher-dot-io#17]
## Add contributor form to the website legesher.io
We want to add a page on the legesher website to thank our contributors for their time, energy, compassion and contribution to the project. (probably should have this renamed as it is connected with the Legesher's GitHub Team `Dream Team` - this team is entirely unrelated). 

When a contributor successfully merges their first pull request, they will get a thank you note + a link to a page to claim their special gift. 

### The Current Situation
_Currently_, I update the link to take the user directly to the typeform to input their information. 

### This Issue Will Solve
_The next phase_ to be solved would be to add a **hidden page** to the website that has the typeform show up when they type in the url `https://legesher.io/dreamteam`. There should not be any direct links to this page on the home page, but rather a quick url I can use to take them directly to the typeform. It should look similar to [this page](https://madi357705.typeform.com/to/wzub4R)

Below is the necessary code to be on `dreamteam.html`: 
``` html
<html>
   <head>
      <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0">
      <title>Dream Team Thank You</title>
      <style type="text/css"> html{ margin: 0; height: 100%; overflow: hidden; } iframe{ position: absolute; left:0; right:0; bottom:0; top:0; border:0; } </style>
   </head>
   <body>
      <iframe id="typeform-full" width="100%" height="100%" frameborder="0" src="https://madi357705.typeform.com/to/wzub4R"></iframe> <script type="text/javascript" src="https://embed.typeform.com/embed.js"></script> 
   </body>
</html>
```

**Comment by marceloFA** [comment]
Can you give more details on how you want this to be done?

**Comment by marceloFA** [comment]
I've added a simple anchor that points to the contributors' page, is this good enough? We can improve on this, if necessary

**Comment by madiedgar** [comment]
you're amazing @marceloFA! I made this as a quick issue to remind myself to do it but your enthusiasm is amazing! I'll update the description with the details. 😄 

Also, make sure to sign up for hacktoberfest, so you can get pull request credit 🎃

**Comment by marissap** [comment]
What about creating another page like _dreamteam.html_ and linking it in the footer. Then every time someone contributes they can make a pull request adding their name to that html page?

We could add that requirement to contributing.md.

<img width="1183" alt="Screen Shot 2019-10-05 at 1 00 24 AM" src="https://user-images.githubusercontent.com/20907831/66250080-927f8d00-e70b-11e9-9a07-f5cb9e57b15c.png">

**Comment by kevinsunny1996** [comment]
Do I need this issue assigned to me before  I work on it ? @madiedgar

**Comment by madiedgar** [comment]
I like that idea a lot @marissap! Want to create another issue and we can hash out the details on that? 😄

**Comment by madiedgar** [comment]
Thanks for your interest @kevinsunny1996! If you want to build on @marceloFA's pull request, that might be most helpful. I added additional information after he created the pull request 🙈

**Comment by madiedgar** [comment]
Updated the information in the issue description @marceloFA, hopefully this clarifies the requirements for the issue 😄 Thank you!!

**Comment by madiedgar** [comment]
Closing this as it is resolved in #44 Thank you everyone!!
